core/modules/system/system.module

```diff
@@ -2030,6 +2030,15 @@ function system_library_info() {
     ),
   );
 
+  // Announce.
+  $libraries['announce'] = array(
+    'title' => 'Announce',
+    'version' => '1.0.0',
+    'js' => array(
+      'core/misc/announce.js' => array(),
```

Comment:
In my testing environment the library was loaded right after "jquery.once.js", even before "backdrop.js" - which caused trouble with "undefined" nagging.

Could `'core/misc/announce.js' => array('group' => JS_DEFAULT),` be a valid solution for that?

Reply:
It was a good solution, thanks.